### ISSUE TITLE
Adjacent links go to the same URL

### ISSUE THREAD

**Issue by Tajana-Somogyi** [ALASdoo/alasdoo_com_v2#30]
Alas logo and Home nav link go to the same destination and are visually close. This is additional navigation and repetition for keyboard and screen reader users.

Make changes:
- [x] No Home link
- [x] Alas logo should have hidden text for screen reader "Alas homepage" (see example)
![Screenshot 2024-11-05 at 12 24 59](https://github.com/user-attachments/assets/c9011643-1b04-43bc-b379-16a4f1d9b847)
